alembic/env.py

```diff
@@ -0,0 +1,84 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from city.models import *
+from temperature.models import *
```

Comment:
Importing all models using '*' is not recommended as it can lead to namespace pollution and make the code harder to understand. It's better to import only the necessary models explicitly.